### DISCUSSION TITLE
Find file / indexing project problem

### DISCUSSION THREAD

**Discussion by silarri** [pulsar-edit/.github#157] [Pulsar Support] [answered]
I have a project opened in Pulsar, but when I try to find a file by name (CTR-T), I get an "Indexing project..." message and a 0 on the right. And I cannot progress from there. I'm not sure if the problem is that it thinks that there are 0 files in the project (which is not correct, as I have a folder added, with subfolders and files inside, that I can see on the navigation left pane) or it is really still indexing.

I have just migrated from Atom and with the last Atom version I have no problem with this.

Any idea is welcome.

**Comment by Daeraxa** [comment, marked as answer]
I think this is a bug with fuzzy finder, we have an existing open issue for it here - https://github.com/pulsar-edit/pulsar/issues/329.

Are you on Windows?

**Reply by confused-Techie** [reply]
Gonna mark the above as an answer here, since we already have an issue tracker for this exact issue. But thanks for reporting it

**Comment by silarri** [comment]
Thank you for pointing this out! Yes, I am on Windows.
